From 773312afa74b1c182ab0ed98300356606026842d Mon Sep 17 00:00:00 2001
From: Matt Turner <mattst88@gmail.com>
Date: Tue, 30 Nov 2010 20:00:22 -0500
Subject: [PATCH] MIPS: delete getpagesize.c

MIPS' getpagesize.c just included ia64's getpagesize.c, which was in
turn just the implementation sysdeps/unix/sysv/linux/getpagesize.c with
ifdef __ASSUME_AT_PAGESIZE which is defined on all Linux platforms since
2.4.0-pre. Just use the generic implementation which will do the exact
same thing.

Signed-off-by: Matt Turner <mattst88@gmail.com>
---
diff --git a/sysdeps/unix/sysv/linux/mips/getpagesize.c b/sysdeps/unix/sysv/linux/mips/getpagesize.c
deleted file mode 100644
index 49492e5..0000000
--- ports/sysdeps/unix/sysv/linux/mips/getpagesize.c
+++ /dev/null
@@ -1 +0,0 @@
-#include <sysdeps/unix/sysv/linux/ia64/getpagesize.c>
-- 
1.7.2.2

